pkg/ct/monitor.go

```diff
@@ -32,22 +35,44 @@ func GetCTLogEntries(logClient *ctclient.LogClient, startIndex int, endIndex int
 	return entries, nil
 }
 
-func ScanEntrySubject(logEntry ct.LogEntry, monitoredSubjects []string) ([]*identity.LogEntry, error) {
+func ScanEntryCertSubject(logEntry ct.LogEntry, monitoredSubjects []string) ([]*identity.LogEntry, error) {
 	subject := logEntry.X509Cert.Subject.String()
-	foundEntries := []*identity.LogEntry{}
+	matchedEntries := []*identity.LogEntry{}
 	for _, monitoredSub := range monitoredSubjects {
 		regex, err := regexp.Compile(monitoredSub)
```

Comment:
This is jumping ahead by one PR, but I think you'll want to match on `CertificateIdentities` rather than `Subjects` since we'd want to match on issuer as well.

Reply:
going to address this in a follow-up PR